src/lib/templateengine.js

```diff
@@ -402,10 +404,10 @@ function TemplateEngine( undefined ) {
   };
   
   this.addAddress = function( address, id ) {
-    if( address in ga_list )
-      ga_list[ address ].push( id );
-    else
+    if ( !(address in ga_list) )
       ga_list[ address ] = [ id ];
+    else if ( $.inArray(id,ga_list[address]) == -1 )
+      ga_list[ address ].push( id );
   };
```

Comment:
What is the reason for this change? Replacing a native JS function by an indirect jQuery function seems like adding only overhead